src/HonzaBotner.Discord/DiscordBot.cs

```diff
@@ -161,5 +176,19 @@ private Task Client_ChannelCreated(DiscordClient client, ChannelCreateEventArgs
         {
             return _eventHandler.Handle(args);
         }
+
+        private async Task ReportException(DiscordGuild guild, string source, Exception exception)
+        {
+            ulong logChannelId = _discordOptions.Value.LogChannelId;
+
+            if (logChannelId == default)
+            {
+                return;
+            }
+
+            DiscordChannel channel = guild.GetChannel(logChannelId);
```

Comment:
`GetChannel` could fail, maybe trycatch it?

Reply:
This is error handling, if we fail here we are doomed anyway

Reply:
![Hail satan elmo](https://c.tenor.com/HC2Sa7-gBIoAAAAM/fire-eyes.gif)

Reply:
Couldn't that also throw an exception, ... ? Like you would trigger another call of this function and again, again, ...😂